.github/workflows/update-dependencies.yml

```diff
@@ -122,6 +122,29 @@ jobs:
               cd -
               rm -rf "${podtmp}"
             fi
+
+            # Download the zip distribution and get the Swift header files.
+            core_version=$(grep "pod 'Firebase/Core'" ios_pod/Podfile | sed "s/.*'\([0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*\)'.*/\1/")
+            echo "Getting Swift header files from Firebase iOS SDK ${core_version} zip distribution"
+            ziptmp="$(mktemp -d)"
+            curl -H 'Authorization: token ${{ github.token }}' "https://github.com/firebase/firebase-ios-sdk/releases/download/v${core_version}/Firebase.zip" -o "${ziptmp}/Firebase.zip"
+            cd "${ziptmp}"
+            echo "Unzipping..."
+            unzip -q Firebase.zip '*-Swift.h'
+            cd -
+            # Copy all *-Swift.h header files into ios_pod/swift_headers/
+            echo Copying headers..."
+            find "${ziptmp}" -name '*-Swift.h' -print0 | xargs -0 -n 1 -J REPLACETEXT cp -f REPLACETEXT ios_pod/swift_headers/
+            copyright_line="// Copyright $(date +%Y) Google LLC"
```

Comment:
Do we need to inject any sort of license disclosure?

Reply:
I'm hesitant to say that we should because these files are generated by Xcode. Since the iOS SDK team doesn't bother, I say we shouldn't.

Reply:
Ok